seldon/seldon-core-operator/base/application.yaml

```diff
@@ -0,0 +1,32 @@
+apiVersion: app.k8s.io/v1beta1
```

Comment:
this should be moved to seldon/seldon-core-operator/overlays/application/

jupyter/notebook-controller has an example including setting commonLabels in the kustomization.yaml.
All labels and matchLabels should be deleted and replaced by the commonLabels in seldon/seldon-core-operator/overlays/application/kustomization.yaml